SocketUtils.cpp

```diff
@@ -44,6 +44,21 @@ bool SocketUtils::getNameInfo(const struct sockaddr *sa, socklen_t salen,
   return true;
 }
 
+int SocketUtils::getAddrFamily(const char *addr) {
+  struct addrinfo hint, *info = 0;
+  memset(&hint, 0, sizeof(hint));
+  hint.ai_family = AF_UNSPEC;
+  hint.ai_flags = AI_NUMERICHOST;
+  int ret = getaddrinfo(addr, 0, &hint, &info);
+  if (ret) {
+    PLOG(ERROR) << "Couldn't get the addr info for " << addr;
```

Comment:
I wrote:

[...] I think we have proper usage of getaddrinfo elsewhere [...]

 @nikunjy  done

nope...

https://github.com/facebook/wdt/blob/master/ServerSocket.cpp#L132